### ISSUE TITLE
Wi-fi signal strength glyphs

### ISSUE THREAD

**Issue by davidzchen** [fabrizioschiavi/pragmatapro#259]
Pragmata Pro contains numerous characters for battery indicators and weather icons under the Private Use Area. Would it be possible to add glyphs for signal strength, which can be used for wi-fi indicators in the terminal, such as with gmoe/tmux-wifi?

**Comment by mfocko** [comment]
Just out of interest, have you tried patching Pragmata to Nerd Font? They have a lot of the icons, I would suggest using the `--careful` switch to avoid conflicts.

https://github.com/ryanoasis/nerd-fonts

**Comment by fabrizioschiavi** [comment]
A Wi-Fi symbol already exists in PragmataPro 0.829
 - U+F1EB
Coming from Nerd Font glyphs
@davidzchen are you interested in multiple glyphs like 1 wave for low quality to 3 or more for high quality of the signal?

**Comment by davidzchen** [comment]
Yes, multiple glyphs for different signal strengths is exactly what I am looking for 🙂.

**Comment by fabrizioschiavi** [comment]
Done. Available from v_0.9

added Wi-fi signal strength glyphs at these codepoints:
󠀀󿿰 U+FFFF0, 󿿱 U+FFFF1, 󿿲 U+FFFF2, 󿿳 U+FFFF3
the strokes weight change if Regular or Bold.

![Image](https://github.com/user-attachments/assets/6dd9a5bb-d321-4662-a717-1051a76f4a24)
